test/FunctionalTests/Tests/DataServices/UnitTests/DesignT4UnitTests/ODataT4CodeGeneratorTests.cs

```diff
@@ -113,6 +131,17 @@ public void CodeGenSimpleEdmx()
             ODataT4CodeGeneratorTestDescriptors.Simple.Verify(code, false/*isCSharp*/, true/*useDSC*/);
         }
 
+        [TestMethod]
+        public void CodeGenSimpleEdmxMultipleFiles()
+        {
+            SplitFiles = true;

```

Comment:
I think `SplitFiles` should be passed as a config option to the code generator like the way the other config options are handled. What's the purpose for setting is a class member instead of a function parameter?

Reply:
True

Reply:
Fixed

Reply:
I also think it ought to be refactored, I find that the current pattern (adding more optional parameters to the `.Verify()` method) is not sustainable, or readable. I think passing an on options object with all the relevant options explicitly set would be better. But I think that's the subject of a separate discussion/PR.